src/main/java/de/hysky/skyblocker/skyblock/dungeon/secrets/Room.java

```diff
@@ -43,10 +48,14 @@
 public class Room {
     private static final Pattern SECRET_INDEX = Pattern.compile("^(\\d+)");
     private static final Pattern SECRETS = Pattern.compile("§7(\\d{1,2})/(\\d{1,2}) Secrets");
+    private static final Vec3d DOOR_SIZE = new Vec3d(3, 4, 3);
+    private static final float[] RED_COLOR_COMPONENTS = DyeColor.RED.getColorComponents();
+    private static final float[] GREEN_COLOR_COMPONENTS = DyeColor.GREEN.getColorComponents();
```

Comment:
These should just be plain 1, 0, 0 and 0, 1, 0 like BetterMap, having it be slightly faded is also weird to see.